docs/transactions.md

```diff
@@ -1198,6 +1198,103 @@ Adds a new Validator node.
 - CLI command:
   - `dcld tx validator add-node --pubkey=<protobuf JSON encoded> --moniker=<string> --from=<account>`
 
+#### DISABLE_VALIDATOR_NODE
+
+**Status: Implemented**
+
+Disables the Validator node (removes from the validator set) by the owner.
+
+- Who can send:
+  - NodeAdmin; owner
+- Parameters:
+  - account: `string` - Bech32 encoded NodeAdmin address or owner account
+- CLI command:
+  - `dcld tx validator disable-node --from=<account>`
+  <br> e.g.:
+    ```
+    dcld query validator disable-node --from=cosmosvaloper1qse069r3w0d82dul4xluqapxfg62qlndsdw9ms
```

Comment:
Do we really support `--from` here in these formats? I think it should be just `dcld tx validator disable-node --from=<account>`...

Reply:
There should be cosmosAddress

Reply:
Done! ✅